script/apply-patches

```diff
@@ -16,18 +16,20 @@ SRC_DIR = os.path.join(SOURCE_ROOT, SRC)
 
 
 def main():
-  error = apply_patches_for_dir(PATCHES_DIR)
-  if not error:
-    if os.environ.has_key('MAS_BUILD'):
-      error = apply_patches_for_dir(PATCHES_MAS_DIR)
-    elif sys.argv[1] == 'mips64el':
-      error = apply_patches_for_dir(PATCHES_MIPS64EL_DIR)
-    if not error:
-      return
+  folders = [PATCHES_DIR]
+  if os.environ.has_key('MAS_BUILD'):
+    folders.append(PATCHES_MAS_DIR)
+  if sys.argv[1] == 'mips64el':
```

Comment:
_(trivial?)_ this is an elif in the previous version. Is it intentional that we could potentially append both MAS and MPS64EL patch directories?

Reply:
> we could potentially append both MAS and MPS64EL patch directories

It won't make any sense, but I think it not that important to be explicitly expressed in the code.

I try to write code in a way the make future diffs clearer and code reviews easier.
Like here, if there were an `elif` and I then someone would like to remove the first condition, they would also have to replace `elif` with `if`. Independent conditional statements make diffs clearer:
```diff
-if sys.platform == 'win32':
-  NINJA = '{0}.exe'.format(NINJA)
-elif IS_MIPS64EL_HOST:
+if IS_MIPS64EL_HOST:
   NINJA = os.path.join(VENDOR_DIR, 'depot_tools', 'ninja-build')
```

```diff
-if sys.platform == 'win32':
-  NINJA = '{0}.exe'.format(NINJA)
 if IS_MIPS64EL_HOST:
   NINJA = os.path.join(VENDOR_DIR, 'depot_tools', 'ninja-build')
```

Unnecessary changes also screw up git annotations.